### DISCUSSION TITLE
GitHub Copilot could not connect to server. Extension activation failed: "unable to verify the first certificate"

### DISCUSSION THREAD

**Discussion by houtian5687** [community/community#14308] [Copilot Conversations] [answered]
GitHub Copilot could not connect to server. Extension activation failed: "unable to verify the first certificate
![屏幕截图(18)](https://user-images.githubusercontent.com/79248318/161684539-e418260f-2b9f-4121-9e44-3e0216dac78f.png)
"

**Comment by davecheney** [comment]
Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

**Reply by houtian5687** [reply]
no

**Comment by lhx2193252482** [comment]
我也是这样

**Comment by Quzeada** [comment]
+1 我也这个问题

**Comment by davecheney** [comment, marked as answer]
Duplicate of https://github.com/github/feedback/discussions/11801
